src/lightning_app/runners/cloud.py

```diff
@@ -556,15 +564,9 @@ def _check_uploaded_folder(root: Path, repo: LocalSourceCodeDir) -> None:
                 f"Your application folder '{root.absolute()}' is more than {CLOUD_UPLOAD_WARNING} MB. "
                 f"The total size is {round(app_folder_size_in_mb, 2)} MB. {len(files)} files were uploaded.\n"
                 + largest_paths_msg
-                + "Perhaps you should try running the app in an empty directory."
+                + "Perhaps you should try running the app in an empty directory.\n"
+                "You can ignore some files or folders by adding them to `.lightningignore`."
```

Comment:
You missed this! Also using `+` as in the rest

```suggestion
                + "You can ignore some files or folders by adding them to `.lightningignore`."
                + " You can also set the `self.lightningingore` attribute in a Flow or Work."
```

Reply:
This was intentional, actually. I like concatenating Python strings without using `+`, but I am happy to follow whatever we do here :)